packages/next/src/server/render.tsx

```diff
@@ -1354,7 +1349,7 @@ export async function renderToHTMLImpl(
       (initialStream: ReactReadableStream, suffix?: string) => {
         return continueFizzStream(initialStream, {
           suffix,
-          inlinedDataStream: serverComponentsInlinedTransformStream?.readable,
```

Comment:
`serverComponentsInlinedTransformStream` is always `null` so TypeScript complained about  unreachable code since `serverComponentsInlinedTransformStream?.readable` is equivalent to `undefined`.